src/parsing/parser.rs

```diff
@@ -5322,6 +5334,7 @@ struct ParseSeparatedValuesOptions<'a> {
     multi_line_options: parser_helpers::MultiLineOptions,
     force_possible_newline_at_start: bool,
     node_sorter: Option<Box<dyn Fn((usize, Option<&Node<'a>>), (usize, Option<&Node<'a>>), &Module<'a>) -> std::cmp::Ordering>>,
+    prefer_inline_multi_line: bool,
```

Comment:
I think this should maybe be `use_new_line_groups` or something related.

Reply:
Got it. It seems much intuitive. I'll change it soon.